src/it/testng/multiple-format/verify.groovy

```diff
@@ -20,7 +20,7 @@ import cucumber.api.testng.AbstractTestNGCucumberTests;
 @CucumberOptions(
         strict = true,
         features = {"${feature1.absolutePath}"},
-        plugin = {"html:${buildDirectory.absolutePath}/cucumber-parallel/1.html", "json:${buildDirectory.absolutePath}/cucumber-parallel/1.json", "pretty"},
+        plugin = {"html:${buildDirectory.absolutePath}/cucumber-parallel/1", "json:${buildDirectory.absolutePath}/cucumber-parallel/1.json", "pretty"},
```

Comment:
The build in html plugin outputs to a directory, not a file. It doesn't need the extension.